### PR TITLE
[#IDEA-146] : Status GO SDK

### PR DESCRIPTION
**Pull request by adriacidre**
## Summary

Provide an easy to use GO-SDK library to allow community developers easily interact with status messaging.

## Swarm Participants

- Lead Contributor:  @adriacidre
- Contributor 1: @pilu
- Contributor 2: (join now)
- Testing & Evaluation:  (join now)

cc: @mandrigin @pilu @antdanchenko @lukaszfryc @arnetheduck

### CODE REVIEW

**Comment by adriacidre** [comment]
Like success metrics, i'll add it to the description.

File: ideas/146-status-go-sdk/README.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+Goal Date: TBD
+
+Description: TBD
+
+Testing Days required: TBD
+
+## Success Metrics
+
+- A dev who is unfamiliar with the SDK is able to set up his environment and build a simple chatbot in under 2 hours, just by following the documentation.
+
```
Comment (oskarth) [inline ideas/146-status-go-sdk/README.md]: Missing exit criteria

**Comment by oskarth** [comment]
Looks good! Approved as draft

File: ideas/146-status-go-sdk/README.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
+## Preamble
+
+    Idea: <to be assigned>
```
Comment (oskarth) [inline ideas/146-status-go-sdk/README.md]: 145-status-go-sdk

**Comment by oskarth** [comment]
Feel free to merge after review comments fixed, at least from my POV

File: README.md [hunk 3]
```diff
@@ -80,7 +81,7 @@ progress again. This ensures the registry is kept up to date.
 | Contributor   |  Ideas |
 |---------------|--------|
 | @anna         |     58, 80, 87, 127 |
-| @adriacidre   | 63     |
+| @adriacidre   | 63, 146     |
```
Comment (oskarth) [inline README.md]: Commitment registry only meant for in progress ideas

**Pull request merged**